src/components/dev-hub/icons/audio-player/fast-forward-icon.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FastForwardIcon = ({ width = '31px', height = '33px', ...props }) => (
```

Comment:
I see the icon on figma has dimensions 28px by 32px. Curious what the difference here is?

Reply:
I don't know why, but if the size is 28x32 the icon will be corrupted. It's minimal size for it

Reply:
okay let's leave it then